### PR TITLE
fix: absolute link in CTA

### PR DESCRIPTION
**Pull request by NaMax66**
Description:
https://virtocommerce.atlassian.net/browse/ST-5110

--

QA-test:

Demo-test:

Download artifact URL: https://vc3prerelease.blob.core.windows.net/packages/vc-theme-b2b-vue-1.41.0-pr-766-392f-392f02e5.zip

### CODE REVIEW

File: client-app/shared/static-content/components/call-to-action.vue [hunk 1]
```diff
@@ -14,15 +19,32 @@
...
+const getLinkAttr = (link?: string): LinkAttrType => {
+  if (link) {
+    if (link.startsWith("/")) {
+      return { to: link };
+    } else {
+      return { externalLink: link };
+    }
+  }
+  return {};
+};
```
Comment (asvishnyakov) [inline client-app/shared/static-content/components/call-to-action.vue:49]: We already have exactly the same code in `call-to-action-with-image`, move shared code to composable please

**Review by asvishnyakov — CHANGES_REQUESTED** [review]

File: content/pages/test_demo_landing.en-US.page [hunk 2]
```diff
@@ -66,7 +66,7 @@
       "type": "call-to-action",
       "id": "calltoactionsNDR",
       "title": "Book a free ecommerce consulting session",
-      "background": null,
+      "background": "bg-gray-800",
```
Comment (goldenmaya) [inline content/pages/test_demo_landing.en-US.page:69]: bad idea to use tailwind classes in json, it should be HEX code, like "\#DDDDDD"

**Review by goldenmaya — CHANGES_REQUESTED** [review]

File: content/pages/test_demo_landing.en-US.page (hunk 2 above)
Comment (NaMax66) [reply content/pages/test_demo_landing.en-US.page:69]: We could select only one of the predefined  in _sections.json

The context is the Page Builder
As a user we can see variants in the settings tab in the preview mode
image

**Review by asvishnyakov — APPROVED** [review]

File: content/pages/test_demo_landing.en-US.page (hunk 2 above)
Comment (goldenmaya) [reply content/pages/test_demo_landing.en-US.page:69]: ok, use new color palette variables then.

**Review by goldenmaya — APPROVED** [review]

**Pull request merged**